src/Form/Extension/DocumentationLinkExtension.php

```diff
@@ -0,0 +1,38 @@
+<?php
+
+/*
+ * This file is part of the Kimai time-tracking app.
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace App\Form\Extension;
+
+use Symfony\Component\Form\AbstractTypeExtension;
+use Symfony\Component\Form\Extension\Core\Type\FormType;
+use Symfony\Component\Form\FormInterface;
+use Symfony\Component\Form\FormView;
+use Symfony\Component\OptionsResolver\OptionsResolver;
+
+/**
+ * Adds a linked help text with the link to the given documentation
+ */
+class DocumentationLinkExtension extends AbstractTypeExtension
+{
+    public function getExtendedType()
+    {
+        return FormType::class;
+    }
+
+    public function buildView(FormView $view, FormInterface $form, array $options)
+    {
+        $view->vars['documentation'] = $options['documentation'] ?? '';
+    }
+
+    public function configureOptions(OptionsResolver $resolver)
```

Comment:
do we want to include docheader everywhere or not?

Reply:
no, the documentation link will only be displayed where it has an value.

Reply:
I think you misunderstood me. I mean the code documentation above the method.

Reply:
Yes I did indeed ;-) I'll update the PR later, thanks for the check!
We need to improve the Scrutinizer and phpcs style checks, see #158 to find such stuff automatically.

Reply:
added the phpdoc